data/assets/css/pages/_communities.scss

```diff
@@ -0,0 +1,115 @@
+#svg-icons {
+  display: none;
+}
+
+#communities {
+  .communities-summary {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .community-summary {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 100%;
+
+    @include size-medium {
+      width: 1/2 * 100%;
+      padding-right: $space-medium;
+    }
+
+    @include size-large {
+      width: 1/3 * 100%;
+      padding-right: $space-small;
+    }
+
+    hr {
+      width: 30%;
+      height: 2px;
+      margin: 1rem auto 2rem;
+      color: #000;
+      background-color: #000;
+
+      @include size-medium {
+        margin: 1rem auto 3.75rem;
+      }
+    }
+  }
+
+  .media {
+    margin-bottom: $space-small;
+  }
+
+  .media-figure {
+    max-width: 50px;
+    margin-bottom: $space-small;
+
+    @include size-small {
+      max-width: 80px;
```

Comment:
why do we need this breakpoint? why not leave the general rule that is "small screens up to medium" with 50px or 80px? (whichever looks good)

Reply:
I think I did it to adjust the layout between the smallest device to `size-medium` and it wasn't finally necessary. Removed!